openspec/changes/archive/2026-04-23-rook-600-acceptance-regression-matrix/specs/gateway/spec.md

```
@@ -0,0 +1,153 @@
+# Delta for Gateway
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)